README.md

```diff
@@ -1,4 +1,4 @@
-# Pseudolocalization [![Version][gem]][gem_url] [![Build Status](https://travis-ci.org/Shopify/pseudolocalization.svg?branch=master)](https://travis-ci.org/Shopify/pseudolocalization)
+# Pseudolocalization [![Version][gem]][gem_url] [![Build Status](https://github.com/Shopify/pseudolocalization/workflows/CI/badge.svg?branch=master)](https://github.com/Shopify/pseudolocalization/actions?query=workflow%3ACI)
```

Comment:
Confirmed this will work as desired once merged in, once master has the new CI code.
<img width="1265" alt="Screen Shot 2021-01-14 at 15 02 28" src="https://user-images.githubusercontent.com/1557529/104551116-98d41700-5679-11eb-95e4-9c95042bd8fd.png">